README.md

```diff
@@ -1,208 +1,143 @@
-# ForgeRock JavaScript SDK
+[![npm (scoped)](https://img.shields.io/npm/v/@forgerock/javascript-sdk?color=%23f46200&label=Version&style=flat-square)](CHANGELOG.md)
 
-The ForgeRock JavaScript SDK is a toolkit that allows web developers to easily integrate intelligent authentication using ForgeRock OpenAM and/or ForgeRock Identity Cloud.
+
+<p align="center">
+  <a href="https://github.com/ForgeRock">
+    <img src="https://www.forgerock.com/themes/custom/forgerock/images/fr-logo-horz-color.svg" alt="Logo">
+  </a>
+  <h2 align="center">ForgeRock SDK for JavaScript</h2>
+  <p align="center">
+    <a href="./blob/master/CHANGELOG.md">Change Log</a>
+    ·
+    <a href="#support">Support</a>
+    ·
+    <a href="#documentation" target="_blank">Docs</a>
+  </p>
+  <hr/>
+</p>
+
+The ForgeRock JavaScript SDK enables you to quickly integrate the [ForgeRock Identity Platform](https://www.forgerock.com/digital-identity-and-access-management-platform) into your JavaScript apps.
+
+Use the SDKs to leverage _[Intelligent Authentication](https://www.forgerock.com/platform/access-management/intelligent-authentication)_ in [ForgeRock's Access Management (AM)](https://www.forgerock.com/platform/access-management) product, to easily step through each stage of an authentication tree by using callbacks.
+
+<!------------------------------------------------------------------------------------------------------------------------------------>
+<!-- REQUIREMENTS - Supported AM versions, API versions, any other requirements. -->
+
+## Requirements
+
+* ForgeRock Identity Platform
+    * Access Management (AM) 6.5.2+
+
+* Browsers:
+    * Chrome 42
+    * Firefox 39
+    * Safari 10.1
+    * Edge 15
+
+> **Tip**: Older browsers can be supported by using polyfills.
```

Comment:
I'm working on a document now that goes into detail regarding this that I believe we should move to official SDK docs sometime soon.

Reply:
Great! Drop me a Jira (or reassign your's) when it's ready and I'll put it on the site somewhere.